src/libs/TransactionUtils.ts

```diff
@@ -578,12 +578,13 @@ function getRecentTransactions(transactions: Record<string, string>, size = 2):
 /**
  * Check if transaction is on hold
  */
-function isOnHold(transaction: OnyxEntry<Transaction>): boolean {
-    if (!transaction) {
+function isOnHold(transactionOrID: OnyxEntry<Transaction> | string): boolean {
```

Comment:
I am not sure if this is an accepted practice to use `transactionOrId`. I think we should have its own util method `isOnHoldByTransactionId`

Reply:
@mountiny Can you confirm?